src/test/java/step3/domain/CarPositionsTest.java

```diff
@@ -0,0 +1,19 @@
+package step3.domain;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarPositionsTest {
+    @Test
+    @DisplayName("CarPositions 객체 생성 후 위치 리스트 크기를 확인한다")
+    void addPosition() {
+        List<Integer> positions = List.of(2, 1, 3);
+        CarPositions carPositions = new CarPositions(positions);
+
+        assertThat(carPositions.getPositions().size()).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(carPositions.getPositions()).hasSize(3);
```
assertj에서 제공하는 기능을 활용해 볼 수 있겠네요!